test/fixtures/organizations.yml

```diff
@@ -15,7 +15,7 @@ organization_two:
   zipcode: t35t
 
 # default organization for unverified staff, dogs do not belong to this org
-organization_three:
+unverified_staff_default:
```

Comment:
Hmmm could we rename this to `default_organization`? It could do more things in the future than just hold a place for staff defaults.

Reply:
Would it be more appropriate to name this `staff_default`? After I set that property (`id: 1`), I think I kind of created a fixture that will act as the default organization that new staff users get assigned to.

Alternatively, it could make more sense to revert this change, and keep a separate organization just for that `id: 1`, like you had before.

Otherwise let me know if you just want to leave this as the "default" organization, I just wanted to bring this up!

Reply:
I think the original naming is fine, too.